### DISCUSSION TITLE
Cannot find camera topic

### DISCUSSION THREAD

**Discussion by chiotisn** [gstavrinos/ai_msc_robotic_systems_assignments#14] [Q&A] [answered]
Hello @gstavrinos, 

Ελπίζω να είναι όλα καλά!

Ακούγεται χαζό, αλλά δεν μπορώ να βρω με τίποτα το topic στο οποίο κάνει publish η κάμερα του robot. 
Ρώτησα τους συμφοιτητές και μου είπαν ότι το topic name μοιάζει με head_front_camera, όμως τίποτα τέτοιο δεν υπάρχει. 
Το μόνο που έχω βρει είναι ένα topic /intel_realsense_r200_depth/image_raw, όμως μόνο αφού ενεργοποιήσω από το RViz το Realsense:
![image](https://github.com/gstavrinos/ai_msc_robotic_systems_assignments/assets/22836597/213459fa-90fb-4cb8-bc29-5e564190678a)

Αυτό είναι το σωστό topic; Αν ναι, πως μπορώ να το ενεργοποιήσω χωρίς το UI από το RViz;

Ευχαριστώ πολύ!

**Comment by gstavrinos** [comment]
Για αρχή, πες μου τί τρέχεις για να ξεκινήσεις την εργασία, και το αποτέλεσμα του `ros2 topic list` ενώ τρέχει το simulation.

**Reply by chiotisn** [reply]
Για να τρέξει η εργασία: `ros2 launch tt_umpire sim_tt.launch.py assignment:=1`
Τα αποτελέσματα του` ros2 topic list`:
[topic_list.txt](https://github.com/gstavrinos/ai_msc_robotic_systems_assignments/files/11781025/topic_list.txt)

Thank you!!

**Reply by chiotisn** [reply]
Το RViz φαίνεται κάπως έτσι:
![image](https://github.com/gstavrinos/ai_msc_robotic_systems_assignments/assets/22836597/d1c79484-fe26-4ca0-b541-9c1b5e61e3ac)

Και αφού ενεργοποιήσω το Realsense , εμφανίζονται 2 νέα topics:

- /intel_realsense_r200_depth/image_raw
- /intel_realsense_r200_depth/points

**Reply by gstavrinos** [reply, marked as answer]
Σύμφωνα και με το topic list, λογικά χρειάζεσαι το `/head_front_camera/rgb/image_raw` για rgb, και τα αντίστοιχα για depth και pointcloud.
